invoice_fiscal_position_update/models/account_invoice.py

```diff
@@ -1,60 +1,45 @@
 # -*- coding: utf-8 -*-
-#############################################################################
-#
-#    Invoice Fiscal Position Update module for OpenERP
-#    Copyright (C) 2011-2014 Julius Network Solutions SARL <contact@julius.fr>
-#    Copyright (C) 2014 Akretion (http://www.akretion.com)
-#    @author Mathieu Vatel <mathieu _at_ julius.fr>
-#    @author Alexis de Lattre <alexis.delattre@akretion.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2016 <OCA>
```

Comment:
Please keep the copyright of the original file. The copyright doesn't belong to the OCA.